apps/api/src/app/organization/organization.module.ts

```diff
@@ -37,7 +36,7 @@ const enterpriseImports = (): Array<Type | DynamicModule | Promise<DynamicModule
 };
 
 function getControllers() {
-  if (isClerkEnabled()) {
+  if (process.env.NOVU_ENTERPRISE === 'true' || process.env.CI_EE_TEST === 'true') {
```

Comment:
I used the condition for enterprise we use everywhere across the app for consistency, although having a similar constant somewhere in shared would be a better approach. I didn't want to change it in this PR directly so maybe in another PR.